com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitProperties.hlsl

```diff
@@ -22,6 +22,8 @@ TEXTURE2D(_MaskMap);
 SAMPLER(sampler_MaskMap);
 TEXTURE2D(_BentNormalMap); // Reuse sampler from normal map
 SAMPLER(sampler_BentNormalMap);
+TEXTURE2D(_BentNormalMapOS);
```

Comment:
note: Add the change improvement with screenshots in the what's new doc here and not in the .md: https://docs.google.com/document/d/1PDi1mR9wR4Pl7stN950CWpSopngaujfP9yqfSPG0SwA/edit#